distmv/normal.go

```diff
@@ -354,3 +355,31 @@ func (n *Normal) setSigma() {
 		n.sigma.FromCholesky(&n.chol)
 	})
 }
+
+// TransformNormal transforms the vector src, generated from a multidimensional
+// standard normal, to one generated under the distribution specified by the
+// receiver. If dst is non-nil, the result will be stored into dst, otherwise
+// a new slice will be allocated. TransformNormal will panic if the length of
+// src is not the dimension of the Normal, or if dst is non-nil and len(src) !=
+// len(dst).
+func (n *Normal) TransformNormal(dst, src []float64) []float64 {
```

Comment:
I guess in this case it's unclear. You could write this method outside, if you took your own Cholesky decomposition. It's a lot more efficient to just use the one we already have. Not sure which way that sways in this case. @kortschak ?